src/main/java/com/easypost/net/EasyPostResource.java

```diff
@@ -281,6 +284,11 @@ private static String createQuery(Map<String, Object> params) throws Unsupported
 		return queryStringBuffer.toString();
 	}
 
+	private static JsonObject createJsonObjectFromMap(Map<String, Object> map) {
```

Comment:
It could be, createJsonObjectFromMap() is a generic method, while createBody() is the specific implementation for HTTP requests.
But we could probably just remove the generi createJsonObjectFromMap() because 1) it's probably not right to have a generic method in this file and 2) it's not being used by anything else, so it's actually just adding more lines rather than saving any lines at the moment.